packages/react-vis/tests/components/xy-plot-tests.js

```diff
@@ -162,9 +161,8 @@ test('testing flexible charts', t => {
 test('Render two stacked bar series with a non-stacked line series chart', t => {
   const $ = mount(<MixedStackedChart />);
 
-  const renderedBarsWrapper = $.find(BarSeries);
-  const renderedLineWrapper = $.find(LineSeries);
```

Comment:
Pretty sure this was correct before. Why do we need to change this?

Reply:
The `BarSeries` that the showcase is referencing, is different than the `BarSeries` in the test.

Reply:
fixed the imports and I was able to change this back so that its properly referencing the same instance of the components.